prisma/postgres/schema.prisma

```diff
@@ -174,4 +174,16 @@ model QuestionAttempt {
     user        User         @relation(fields: [userId], references: [id], onDelete: Cascade)
 
     @@unique([userId, questionId, answerId])
+
+}
+
+model SessionMessage {
+    sessionId   String
+    senderId    String
```

Comment:
Shouldn't we bind this to user via a relations

Reply:
done